File: fixture_repo/payments.js
import { formatCurrency, generateId } from './utils';
import { createSession } from './auth';

export function processPayment(userId, amount) {
  const session = createSession(userId);
  const transactionId = generateId();
  const formatted = formatCurrency(amount);
  return { transactionId, formatted, session };
}

export function refundPayment(transactionId) {
  return transactionId;
}

File: fixture_repo_python/payments.py
from .utils import format_currency, generate_id
from .auth import create_session

def process_payment(user_id, amount):
    session = create_session(user_id)
    transaction_id = generate_id()
    formatted = format_currency(amount)
    return {
        "transactionId": transaction_id,
        "formatted": formatted,
        "session": session
    }

def refund_payment(transaction_id):
    return transaction_id
